---
title: "Publications"
layout: gridlay
sitemap: false
permalink: /publications/
---

# Publications

For a full list see: [Google Scholar](https://scholar.google.de/citations?user=xG5HYekAAAAJ&hl=en)  
<sup>\#</sup> Co-first authors; <sup>\*</sup> Co-corresponding authors

### Selected papers

* [Benchmarking computational methods to identify spatially variable genes and peaks](https://www.biorxiv.org/content/10.1101/2023.12.02.569717v1.abstract) \
**Zhijian Li**, Zain M.Patel, Dongyuan Song, Guanao Yan, Jingyi Jessica Li, Luca Pinello. \
Biorxiv(2023). 

* [RGT: a toolbox for the integrative analysis of high throughput regulatory genomics data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05184-5) \
**Zhijian Li<sup>\#\*</sup>**, Chao-Chung Kuo<sup>\#\*</sup>, Fabio Ticconi, Mina Shaigan, Julia Gehrmann, Eduardo Gade Gusmao, Manuel Allhoff, Martin Manolov, Martin Zenke, Ivan G. Costa<sup>\*</sup> \
**BMC Bioinformatics 24**, 79 (2023).

* [scMEGA: Single-cell Multiomic Enhancer-based Gene Regulatory Network Inference](https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbad003/6986159?login=false) \
**Zhijian Li**, James S Nagai, Christoph Kuppe, Rafael Kramann, Ivan G. Costa \
**Bioinformatics Advances** (2023).


* [Spatial multi-omic map of human myocardial infarction](https://www.nature.com/articles/s41586-022-05060-x)\
Christoph Kuppe<sup>\#</sup>, Ricardo O. Ramirez Flores<sup>\#</sup>, **Zhijian Li<sup>\#</sup>**, Sikander Hayat, Rebecca T. Levinson, Xian Liao, Monica T. Hannani, Jovan Tanevski, Florian Wünnemann, James S. Nagai, Maurice Halder, David Schumacher, Sylvia Menzel, Gideon Schäfer, Konrad Hoeft, Mingbo Cheng, Susanne Ziegler, Xiaoting Zhang, Fabian Peisker, Nadine Kaesler, Turgay Saritas, Yaoxian Xu, Astrid Kassner, Jan Gummert, Michiel Morshuis, Junedh Amrute, Rogier J. A. Veltrop, Peter Boor, Karin Klingel, Linda W. Van Laake, Aryan Vink, Remco M. Hoogenboezem, Eric M. J. Bindels, Leon Schurgers, Susanne Sattler, Denis Schapiro, Rebekka K. Schneider, Kory Lavine, Hendrik Milting<sup>\*</sup>, Ivan G. Costa<sup>\*</sup>, Julio Saez-Rodriguez<sup>\*</sup>, Rafael Kramann<sup>\*</sup> \
**Nature 608** 766–777 (2022).

* [Chromatin-accessibility estimation of single-cell ATAC-seq data with scOpen](https://www.nature.com/articles/s41467-021-26530-2)\
**Z Li**<sup>\#</sup>, C Kuppe<sup>\#</sup>, S Ziegler, M Cheng, N Kabgani, S Menzel, M Zenke, R Kramann<sup>\*</sup>, IG Costa<sup>\*</sup>.\
**Nature Communications** (2021).

* [Identification of transcription factor binding sites using ATAC-seq](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1642-2)\
**Z Li**, MH Schulz, T Look, M Begemann, M Zenke, I G Costa.\
**Genome Biology** (2019).

* [Improving the JADE Algorithm by Clustering Successful Parameters](https://www.inderscienceonline.com/doi/abs/10.1504/IJWMC.2016.081159)\
**Z Li**, J Guo, S Yang.\
**International Journal of Wireless and Mobile Computing** (2016).

### Contributed papers

* [Joint genotypic and phenotypic outcome modeling improves base editing variant effect quantification](https://www.nature.com/articles/s41588-024-01726-6) \
Jayoung Ryu, Sam Barkal, Tian Yu, Martin Jankowiak, Yunzhuo Zhou, Matthew Francoeur, Quang Vinh Phan, **Zhijian Li**, Manuel Tognon, Lara Brown, Michael I. Love, Guillaume Lettre, David B. Ascher, Christopher A. Cassa, Richard I. Sherwood, Luca Pinello
**Nature Genetics** (2024).


* [A lncRNA identifies IRF8 enhancer in feedback control of DC development](https://elifesciences.org/articles/83342)\
H Xu, **Z Li**, CC Kuo, K Götz, T Look, MAS de Toledo, K Seré, IG Costa, M Zenke.\
**eLife** (2023).

* [Adult human kidney organoids originate from CD24+ cells and represent an advanced model for adult polycystic kidney disease](https://www.nature.com/articles/s41588-022-01202-z)\
Y Xu<sup>\#</sup>, C Kuppe<sup>\#</sup>, JP Patón, S Hayat, J Kranz,  AT Abdallah, J Nagai, **Z Li**, F Peisker, T Saritas, M Halder, S Menzel, K Hoeft, A Kenter, H Kim, C van Roeyen, M Lehrke, J Moellmann, T Speer, EM Buhl, R Hoogenboezem, P Boor, J Jansen, C Knopp, I Kurth, B Smeets, E Bindels, M Reinders, C Baan, J Gribnau, E Hoorn, J Steffens, T Huber, IG Costa, J Floege, RK Schneider, J Saez-Rodriguez, BS Freedman<sup>\*</sup>, R Kramann<sup>\*</sup>.\
**Nature Genetics** (2022).

* [MOJITOO: a fast and universal method for integration of multimodal single cell data](https://academic.oup.com/bioinformatics/article/38/Supplement_1/i282/6617520)\
M Cheng, **Z Li**, IG Costa.\
**Bioinformatics** (2022).

* [Transcriptional changes and the role of ONECUT1 in hPSC pancreatic differentiation](https://www.nature.com/articles/s42003-021-02818-3)\
S Heller, **Z Li**, Q Lin, R Geusz, M Breunig, M Hohwieler, X Zhang, G Nair, T Seufferlein, M Hebrok, M Sander, C Julier, A Kleger<sup>\*</sup>, IG Costa<sup>\*</sup>.\
**Communications Biology** (2021).

* [Mutations and variants of ONECUT1 in diabetes](https://www.nature.com/articles/s41591-021-01502-7)\
A Philippi<sup>\#</sup>, S Heller<sup>\#</sup>, IG Costa<sup>\#</sup>, V Senée<sup>\#</sup>, M Breunig, **Z Li**, G Kwon, R Russell, A Illing, Q Lin, M Hohwieler, A Degavre, P Zalloua, S Liebau, M Schuster, J Krumm, X Zhang, R Geusz, JR Benthuysen, A Wang, J Chiou, K Gaulton, H Neubauer, E Simon, T Klein, M Wagner, G Nair, C Besse, CD Roulland, R Olaso, J Deleuze, B Kuster, M Hebrok, T Seufferlein, M Sander, BO Boehm, F Oswald, M Nicolino<sup>\*</sup>, C Julier<sup>\*</sup>, A Kleger<sup>\*</sup>.\
**Nature Medicine** (2021).

* [Accelerating differential evolution based on a subset-to-subset survivor selection operator](https://link.springer.com/article/10.1007/s00500-018-3060-x)\
J Guo, **Z Li**, S Yang.\
**Soft Computing** (2019).

* [Dissipative differential evolution with self-adaptive control parameters](https://ieeexplore.ieee.org/abstract/document/7257274)\
J Guo, **Z Li**, W Xie, H Wang.\
**IEEE Congress on Evolutionary Computation** (2015).
