---
title: "Publications"
layout: gridlay
sitemap: false
permalink: /publications/
---

# Publications

For a full list see: [Google Scholar](https://scholar.google.de/citations?user=xG5HYekAAAAJ&hl=en)  
<sup>\#</sup> Co-first authors; <sup>\*</sup> Co-corresponding authors

### First/co-first author papers

* C. Kuppe<sup>\#</sup>, R. Flores<sup>\#</sup>, **Z. Li**<sup>\#</sup>, S. Hayat, M. Hannani, J. Tanevski, M. Halder, M. Cheng, S. Ziegler, X. Zhang, F. Preisker, N. Kaesler, Y. Xu, R. M. Hoogenboezem, E. M. Bindels, R. K. Schneider, H. Milting<sup>\*</sup>, I. G. Costa<sup>\*</sup>, J. S. Rodriguez<sup>\*</sup>, R. Kramann<sup>\*</sup>. Spatial multi-omic map of human myocardial infarction. **Nature** (2022). (Accepted)

* **Z. Li**<sup>\#</sup>, C. Kuppe<sup>\#</sup>, S. Ziegler, M. Cheng, N. Kabgani, S. Menzel, M. Zenke, R. Kramann<sup>\*</sup>, I. G. Costa<sup>\*</sup>. Chromatin-accessibility estimation of single-cell ATAC-seq data with scOpen. **Nature Communications** (2021).

* **Z. Li**, M. H. Schulz, T. Look, M. Begemann, M. Zenke, I. G. Costa. Identification of transcription factor binding sites using ATAC-seq. **Genome Biology** (2019).

* **Z. Li**, J. Guo, S. Yang. Improving the JADE Algorithm by Clustering Successful Parameters.
**International Journal of Wireless and Mobile Computing** (2016).

### Contributed papers

* K. Ohl, J. Hriczko, S. Schulz, T. Look, T. Goodarzi, S.H. Subramanyam, T. Clarner, M. Scheld, M. Kipp, E. Verjans, S. Boll, I. G. Costa, **Z. Li**, L. Gan, B. Denecke, A. Schippers, S. Floess, J. Huehn, E. Schmitt, T. Bopp, R. Beyaert, B. Lambrecht, M. Zenke, N. Wagner, K. Tenbrock. Foxp3-specific deletion of CREB generates Th2 biased ST-2 positive regulatory T-cells with enhanced IL-10 production and suppressive capacity. (2022) (Submitted).

* E. F. Brandt, T. H. Wirtz, **Z. Li**, M. M. Ibrahim, P. Fischer, A. Beckers, A. Flabhove, I. G. Costa, R. Kramann, E. van der Vorst, C. Liedtke, H. Sahin, C. Trautwein, M. L. Berres. CXCR3-expressing macrophages regulate angiogenesis, tumor cell proliferation and immune evasion in hepatocellular carcinoma. (2022) (Submitted).

* H. Xu, M. A. Karpinska, **Z. Li**, C. Kuo, K. Götz, T. Look, M. de Toledo, K. Seré, G.Z. Bensch, A. Oudelaar, I. G. Costa and M. Zenke. A lncRNA identifies IRF8 enhancer in feedback control of DC development. **Molecular Cell** (2022). (Submitted)

* D. Schumacher<sup>\#</sup>, M. Cheng<sup>\#</sup>, C. Kuppe, E. M. Bindels, R. K. Schneider, **Z. Li**, I. G. Costa<sup>*</sup>, R. Kramann<sup>*</sup>. Single Nucleus ATAC Sequencing Reveals Epigenomic Regulation After Myocardial Infarction. **Nature Cardiovascular Research** (2022). (Submitted)

* M. Cheng, **Z. Li**, I. G. Costa. MOJITOO: a fast and universal method for integration of multimodal single cell data. **Bioinforamtics** (2022).

* S. Heller, **Z. Li**, Q. Lin, R. Geusz, M. Breunig, M. Hohwieler, X. Zhang, G. Nair, T. Seufferlein, M. Hebrok, M. Sander, C. Julier, A. Kleger<sup>*</sup>, I. G. Costa<sup>*</sup>. Transcriptional changes and the role of ONECUT1 in hPSC pancreatic differentiation. **Communications Biology** (2021).

* A. Philippi<sup>\#</sup>, S. Heller<sup>\#</sup>, I.G. Costa<sup>\#</sup>, V. Senée<sup>\#</sup>, M. Breunig, **Z. Li**, G. Kwon, R. Russell, A. Illing, Q. Lin, M. Hohwieler, A. Degavre, P. Zalloua, S. Liebau, M. Schuster, J. Krumm, X. Zhang, R. Geusz, J.R. Benthuysen, A. Wang, J. Chiou, K. Gaulton, H. Neubauer, E. Simon, T. Klein, M. Wagner, G. Nair, C. Besse, C. Dandine-Roulland, R. Olaso, J. Deleuze, B. Kuster, M. Hebrok, T. Seufferlein, M. Sander, B.O. Boehm, F. Oswald, M. Nicolino<sup>*</sup>, C. Julier<sup>*</sup>, A. Kleger<sup>*</sup>. Mutations and variants of ONECUT1 in diabetes. **Nature Medicine** (2021).

* J. Guo, **Z. Li**, S. Yang. Accelerating differential evolution based on a subset-to-subset survivor selection operator. **Soft Computing** (2019).

* J. Guo, W. Xie, **Z, Li**. Enhancing JADE utilizing distance of best solutions. **Computer Engineering and Design** (2018).

* J. Guo, **Z. Li**, W. Xie, H. Wang. Dissipative differential evolution with self-adaptive control parameters. **IEEE Congress on Evolutionary Computation** (2015).
