---
title: "Publications"
layout: gridlay
sitemap: false
permalink: /publications/
---

# Publications

For a full list see: [Google Scholar](https://scholar.google.de/citations?user=xG5HYekAAAAJ&hl=en)  
<sup>\#</sup> Co-first authors; <sup>\*</sup> Co-corresponding authors

### First/co-first author papers

* **Z Li<sup>\#</sup>**, C C Kuo<sup>\#</sup>, F Ticconi, M Shaigan, E G Gusmao, M Allhoff, M Manolov, M Zenke, I G Costa. RGT: a toolbox for the integrative analysis of high throughput regulatory genomics data. (2022) (***Submitted***)

* C Kuppe<sup>\#</sup>, **Z Li**<sup>\#</sup>, R T Levinson<sup>\#</sup>, P B Mompel<sup>\#</sup>, T Lu, R Flores, M Halder, M Cheng, X Zhang, R K Schneider, R Thadhani, S Hayat, K Lim, I G Costa<sup>\*</sup>, J S Rodriguez<sup>\*</sup>, R Kramann<sup>\*</sup>. Integrative single-cell multi-omics analysis reveals traits associated with early heart failure. (2022) (***Submitted***)

* **Z Li**, J Nagai, C Kuppe, R Kramann, I G Costa. [scMEGA: Single-cell Multiomic Enhancer-based Gene Regulatory Network Inference](https://www.biorxiv.org/content/10.1101/2022.08.10.503335v1). ***BioRxiv*** (2022).

* C Kuppe<sup>\#</sup>, ROR Flores<sup>\#</sup>, **Z Li<sup>\#</sup>**, S Hayat, RT Levinson, X Liao, MT Hannani, J Tanevski, F Wünnemann, J Nagai, M Halder, D Schumacher, S Menzel, G Schäfer, K Hoeft, M Cheng, S Ziegler, X Zhang, F Peisker, N Kaesler, T Saritas, Y Xu, A Kassner, J Gummert, M Morshuis, J Amrute, R J A Veltrop, P Boor, K Klingel, LW Van Laake, A Vink, RM Hoogenboezem18, E M J Bindels, L Schurgers, S Sattler, D Schapiro, RK Schneider, K Lavine, H Milting<sup>\*</sup>, IG Costa<sup>\*</sup>, J Saez-Rodriguez<sup>\*</sup> and R Kramann<sup>\*</sup>. [Spatial multi-omic map of human myocardial infarction](https://www.nature.com/articles/s41586-022-05060-x). ***Nature*** (2022).

* **Z Li**<sup>\#</sup>, C Kuppe<sup>\#</sup>, S Ziegler, M Cheng, N Kabgani, S Menzel, M Zenke, R Kramann<sup>\*</sup>, IG Costa<sup>\*</sup>. [Chromatin-accessibility estimation of single-cell ATAC-seq data with scOpen](https://www.nature.com/articles/s41467-021-26530-2). ***Nature Communications*** (2021).

* **Z Li**, MH Schulz, T Look, M Begemann, M Zenke, I G Costa. [Identification of transcription factor binding sites using ATAC-seq](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1642-2). ***Genome Biology*** (2019).

* **Z Li**, J Guo, S Yang. [Improving the JADE Algorithm by Clustering Successful Parameters](https://www.inderscienceonline.com/doi/abs/10.1504/IJWMC.2016.081159).
***International Journal of Wireless and Mobile Computing*** (2016).

### Contributed papers
<!-- * K. Ohl, J. Hriczko, S. Schulz, T. Look, T. Goodarzi, S.H. Subramanyam, T. Clarner, M. Scheld, M. Kipp, E. Verjans, S. Boll, I. G. Costa, **Z. Li**, L. Gan, B. Denecke, A. Schippers, S. Floess, J. Huehn, E. Schmitt, T. Bopp, R. Beyaert, B. Lambrecht, M. Zenke, N. Wagner, K. Tenbrock. Foxp3-specific deletion of CREB generates Th2 biased ST-2 positive regulatory T-cells with enhanced IL-10 production and suppressive capacity. (2022) (Submitted).

* E. F. Brandt, T. H. Wirtz, **Z. Li**, M. M. Ibrahim, P. Fischer, A. Beckers, A. Flabhove, I. G. Costa, R. Kramann, E. van der Vorst, C. Liedtke, H. Sahin, C. Trautwein, M. L. Berres. CXCR3-expressing macrophages regulate angiogenesis, tumor cell proliferation and immune evasion in hepatocellular carcinoma. (2022) (Submitted).

* D. Schumacher<sup>\#</sup>, M. Cheng<sup>\#</sup>, C. Kuppe, E. M. Bindels, R. K. Schneider, **Z. Li**, I. G. Costa<sup>*</sup>, R. Kramann<sup>*</sup>. Single Nucleus ATAC Sequencing Reveals Epigenomic Regulation After Myocardial Infarction. (2022). (Submitted) -->

* H Xu, **Z Li**, CC Kuo, K Götz, T Look, MAS de Toledo, K Seré, IG Costa and M Zenke. [A lncRNA identifies IRF8 enhancer in feedback control of DC development](https://www.biorxiv.org/content/10.1101/2022.08.11.503623v1). ***BioRxiv***. (2022).

* M Cheng, **Z Li**, I G Costa. [MOJITOO: a fast and universal method for integration of multimodal single cell data](https://academic.oup.com/bioinformatics/article/38/Supplement_1/i282/6617520). ***Bioinforamtics*** (2022).

* S Heller, **Z Li**, Q Lin, R Geusz, M Breunig, M Hohwieler, X Zhang, G Nair, T Seufferlein, M Hebrok, M Sander, C Julier, A Kleger<sup>*</sup>, I G Costa<sup>*</sup>. [Transcriptional changes and the role of ONECUT1 in hPSC pancreatic differentiation](https://www.nature.com/articles/s42003-021-02818-3). ***Communications Biology*** (2021).

* A Philippi<sup>\#</sup>, S Heller<sup>\#</sup>, IG Costa<sup>\#</sup>, V Senée<sup>\#</sup>, M Breunig, **Z Li**, G Kwon, R Russell, A Illing, Q Lin, M Hohwieler, A Degavre, P Zalloua, S Liebau, M Schuster, J Krumm, X Zhang, R Geusz, J.R Benthuysen, A Wang, J Chiou, K Gaulton, H Neubauer, E Simon, T Klein, M Wagner, G Nair, C Besse, C Dandine-Roulland, R Olaso, J Deleuze, B Kuster, M Hebrok, T Seufferlein, M Sander, BO Boehm, F Oswald, M Nicolino<sup>*</sup>, C Julier<sup>*</sup>, A Kleger<sup>*</sup>. [Mutations and variants of ONECUT1 in diabetes](https://www.nature.com/articles/s41591-021-01502-7). ***Nature Medicine*** (2021).

* [Accelerating differential evolution based on a subset-to-subset survivor selection operator](https://link.springer.com/article/10.1007/s00500-018-3060-x). \
  J Guo, **Z Li**, S Yang. \
  **Soft Computing** (2019).

* [Dissipative differential evolution with self-adaptive control parameters](https://ieeexplore.ieee.org/abstract/document/7257274). \
  J Guo, **Z Li**, W Xie, H Wang.\
  **IEEE Congress on Evolutionary Computation** (2015).
